.vscode/tasks.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Editor specific config shouldn't be added to the repo.

(I have `.vscode` in my user level `~/.gitignore`, but perhaps we should add it to the repo level one as well).